build.assets/docker/os-rootfs/etc/node-problem-detector/docker-monitor.json

```diff
@@ -0,0 +1,18 @@
+{
+  "plugin": "journald",
+  "pluginConfig": {
+    "source": "dockerd"
+  },
+  "logPath": "/var/log/journal",
+  "lookback": "5m",
+  "bufferSize": 10,
+  "source": "docker-monitor",
+  "conditions": [],
```

Comment:
nit: is empty conditions still a requirement for specification?

Reply:
No idea, this is default configuration. I don't think it hurts anybody.

Reply:
The checks/conditions are from the "default" configs but I had to tweak some paths for the planet environment and remove some checks not applicable to us. I got a feeling that their "default" [config](https://github.com/kubernetes/node-problem-detector/tree/master/config) is more like a "sample" config which you need to adjust to your environment anyway. I guess if/when we update NPD itself we can also look if they added new stuff to their default config. On the upside, it makes it real easy for us to extend it with our own stuff should we need/want to.